iphone/Maps/Core/Location/MWMLocationManager.mm

```diff
@@ -283,8 +274,7 @@ - (void)processLocationStatus:(MWMLocationStatus)locationStatus
 - (void)processHeadingUpdate:(CLHeading *)headingInfo
 {
   self.lastHeadingInfo = headingInfo;
-  self.frameworkUpdateMode |= MWMLocationFrameworkUpdateHeading;
-//  location::CompassInfo const compassInfo = compassInfoFromHeading(headingInfo);
+  GetFramework().OnCompassUpdate(location_util::compassInfoFromHeading(headingInfo));
```

Comment:
Why was it commented before?

Reply:
How the current code was working without this framework call? What was broken and what is fixed now? Won't it influence battery usage?

Reply:
Checked, and this fix should work, thanks! Simple and efficient.